string origin_model_name
string origin_link_name
float32 target_x_coordinate
float32 target_y_coordinate
---
bool ok

